detekt/detekt-config.yml

```diff
@@ -107,6 +107,7 @@ complexity:
   LargeClass:
     active: true
     threshold: 600
+    excludes: 'TLSCertificatePinning.kt'
```

Comment:
```suggestion
    excludes: ['**/TLSCertificatePinning.kt']
```

Reply:
`TLSCertificatePinning.kt` is not in the same directory as the detekt config. We'd need the absolute path or use a ** to look for all directories for the file.

Reply:
Ohh I see, I couldn't figure out why tests were failing yesterday. Appreciate the help!